holoclean.py

```diff
@@ -207,6 +213,17 @@ def __init__(self, env, name="session"):
         :param env: Holoclean environment
         :param name: Name for the Holoclean session
         """
+        # use DEBUG logging level if verbose enabled
+        root_logger = logging.getLogger()
+        gensim_logger = logging.getLogger('gensim')
+        root_level, gensim_level = logging.INFO, logging.WARNING
+        if env['verbose']:
```

Comment:
keep this if statement here, but move everything else to line 12 with the logging details. I understand we need `env` here, so we only modify the logging level if `verbose`